packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -126,6 +128,8 @@ const ModalManager = () => {
 
     return (
         <React.Fragment>
+            {/* TODO: Once everything is merged place this in main component */}
+            <ReadyToUpdateWallets is_eu={is_eu} is_high_risk={is_high_risk} />
```

Comment:
you can get this props in the component itself, don't need to pass them here

Reply:
Done 😃 👍🏼